apps/condo/domains/acquiring/access/Payment.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Generated by `createschema acquiring.Payment 'amount:Decimal; currencyCode:Text; time:DateTimeUtc; accountNumber:Text; purpose?:Text; receipts:Relationship:BillingReceipt:PROTECT; multiPayment:Relationship:MultiPayment:PROTECT; context:Relationship:AcquiringIntegrationContext:PROTECT;' --force`
+ */
+
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+const { AcquiringIntegrationAccessRight } = require('@condo/domains/acquiring/utils/serverSchema')
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadPayments ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.isAdmin || user.isSupport) return {}
+    // User can get only it's own payments
+    if (user.type === RESIDENT) {
+        return {
+            multiPayment: { user: { id: user.id } },
+        }
+    }
+    return {
+        OR: [
+            // Acquiring integration account can see it's payments
+            { context: { integration: { accessRights_some: { user: { id: user.id } } } } },
+            // Employee with `canManagePayments` can see theirs organization payments
+            { context: { organization: { employees_some: { user: { id: user.id }, role: { canManagePayments: true }, deletedAt: null, isBlocked: false } } } },
+        ],
+    }
+}
+
+async function canManagePayments ({ authentication: { item: user }, operation, context }) {
+    if (!user) return throwAuthenticationError()
+    if (user.isAdmin) return true
+    if (operation === 'create') {
+        // Acquiring integration can create payments, linked to own context and own MultiPayments
```

Comment:
you need to check that integration create can't create own Payments

Reply:
Please check validateInput for Payment.multiPayment field